hack/dind-cluster.sh

```diff
@@ -179,12 +179,17 @@ function start() {
   sudo modprobe br_netfilter || true
   sudo sysctl -w net.bridge.bridge-nf-call-iptables=0
   ensure-loopback-for-dind "${NUM_NODES}"
+  if [ ! -d "${CONFIG_ROOT}" ]; then
```

Comment:
this is not needed, the `mkdir -p` will create the dir if doesn't exists or do nothing if it exists.